src/app/code/Komoju/Payments/Api/KomojuApi.php

```diff
@@ -0,0 +1,116 @@
+<?php
```

Comment:
Is this identical to the file from the komoju-php library we have?

Reply:
No, I actually had to rewrite some of it because the magento lint standard doesn't want explicit curl calls. They have their own wrapper to use (`\Magento\Framework\HTTP\Client\Curl`). Also "magento"ifyed it by properly using their dependency injection since i was also prevented from using the `require_once` I was using previously

Reply:
Ok 👍 . I would prefer it if we could use the same library, since that makes updating our API integration a lot easier (since we would just update the package, then update the dependency in each of the PHP plugins), but  I'm not sure :

- If Magento allows third party dependencies in their plugins
- If we have a way to easily host and deploy a third party PHP library

In lieu of those, I think copying the code to use Magento directly is fine, but I'd love to add a comment to the top of the file to explain it's connection to the library (for future maintainers 😄 )